### DISCUSSION TITLE
Got a job mail from notifications@github.com

### DISCUSSION THREAD

**Discussion by DamiAdeeso** [community/community#109384] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I got a job mail from notifications at github.com but when i clicked on the attached link i was redirected to githubtalentcommunity.online and access to my github was requested. Please is this legitimate.

**Comment by BillZong** [comment]
Me, too.
This should be a phishing attack? Just don't get it why the mail address in Github's domain.

**Reply by DamiAdeeso** [reply]
Could it be possible GitHub was compromised cause the email address seems legitimate.

**Reply by somesh-banerjee** [reply]
I also got one.
And I got it as a notification of a PR where I was mentioned.
Looks like someone is misusing the notification feature of GitHub.

**Comment by Soumya6Tiwari** [comment]
@DamiAdeeso  please just dont click on the link , i did and now my profile has been marked as spam and it is also invisible to others , i dont know how to access it . kindly help me

**Comment by somesh-banerjee** [comment, marked as answer]
It's fake 
check this discussion https://github.com/orgs/community/discussions/109171

**Comment by Soumya6Tiwari** [comment]
@somesh-banerjee  I have created the ticket to recover my account ? can you please tell me by when it will be ok, like how much time it will take?

**Comment by ghostinhershell** [comment]
Thanks so much for reporting this @DamiAdeeso. I'm going to copy/paste the response that my collegue gave in the Discussion @somesh-banerjee linked to above for visibility sake:

> We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.
> 
> We would like to bring to our users’ attention to the following:
> - Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
> - [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
> - GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
> - GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
> This spam activity targets and abuses GitHub’s mention and notification functionality.
> - This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by ettaboyle** [comment]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by exSnake** [comment]
I just received a mail from notification@github.com saying to click a link to "scan" my repository cause a potentially issue was detected and the link redirected me to github-scanner.com
